cli/admin/src/test/java/io/pravega/cli/admin/utils/TestUtils.java

```diff
@@ -131,22 +136,24 @@ public static ClusterWrapper createPravegaCluster(boolean authEnabled, boolean t
      * @param containerCount the container count.
      * @param authEnabled whether the cli requires authentication to access the cluster.
      * @param tlsEnabled whether the cli requires TLS to access the cluster.
+     * @param accessTokenTtl how long the access token will last
      */
     @SneakyThrows
     public static AdminCommandState createAdminCLIConfig(String controllerRestUri, String controllerUri, String zkConnectUri,
-                                                         int containerCount, boolean authEnabled, boolean tlsEnabled) {
+                                                         int containerCount, boolean authEnabled, boolean tlsEnabled, Duration accessTokenTtl) {
         AdminCommandState state = new AdminCommandState();
         Properties pravegaProperties = new Properties();
         System.out.println("REST URI: " + controllerRestUri);
         pravegaProperties.setProperty("cli.controller.connect.rest.uri", controllerRestUri);
         pravegaProperties.setProperty("cli.controller.connect.grpc.uri", controllerUri);
         pravegaProperties.setProperty("pravegaservice.zk.connect.uri", zkConnectUri);
         pravegaProperties.setProperty("pravegaservice.container.count", Integer.toString(containerCount));
-        pravegaProperties.setProperty("cli.controller.connect.channel.auth", Boolean.toString(authEnabled));
-        pravegaProperties.setProperty("cli.controller.connect.credentials.username", SecurityConfigDefaults.AUTH_ADMIN_USERNAME);
-        pravegaProperties.setProperty("cli.controller.connect.credentials.pwd", SecurityConfigDefaults.AUTH_ADMIN_PASSWORD);
-        pravegaProperties.setProperty("cli.controller.connect.channel.tls", Boolean.toString(tlsEnabled));
-        pravegaProperties.setProperty("cli.controller.connect.trustStore.location", pathToConfig() + SecurityConfigDefaults.TLS_CA_CERT_FILE_NAME);
+        pravegaProperties.setProperty("cli.channel.auth", Boolean.toString(authEnabled));
+        pravegaProperties.setProperty("cli.credentials.username", SecurityConfigDefaults.AUTH_ADMIN_USERNAME);
+        pravegaProperties.setProperty("cli.credentials.pwd", SecurityConfigDefaults.AUTH_ADMIN_PASSWORD);
+        pravegaProperties.setProperty("cli.channel.tls", Boolean.toString(tlsEnabled));
+        pravegaProperties.setProperty("cli.trustStore.location", pathToConfig() + SecurityConfigDefaults.TLS_CA_CERT_FILE_NAME);
+        pravegaProperties.setProperty("cli.trustStore.access.token.ttl", Long.toString(accessTokenTtl.toSeconds()));
```

Comment:
You are not setting the correct property here. The property, as defined in `CLIConfig` is named `cli.trustStore.access.token.ttl.seconds`.